contrib/lib/src/uuid.rs

```diff
@@ -164,6 +164,12 @@ mod test {
     fn test_from_param_invalid() {
         let uuid_str = "c1aa1e3b-9614-4895-9ebd-705255fa5bc2p";
         let uuid_result = Uuid::from_param(uuid_str.into());
-        assert_eq!(uuid_result, Err(UuidParseError::InvalidLength(37)));
+        assert_eq!(
+            uuid_result,
+            Err(UuidParseError::InvalidLength {
+                expected: uuid_ext::parser::Expected::Any(&[36, 32]),
```

Comment:
A minor version of `uuid` might conceivably change the list of values in `Expected` -- either to include the value 45 for URN format or for a new format it parses. Maybe this assert should be changed or `#[should_panic(expected=...)]` could be used to loosen the expectations for the error.